src/components/input/input.tsx

```diff
@@ -437,6 +441,13 @@ export class Input implements LabelableComponent, FormComponent, InteractiveComp
     });
   };
 
+  private emitChangeEvent = (): void => {
```

Comment:
Is there an alternative name we could use for this? It emits conditionally, so it could be misleading.

Reply:
Yeah, name for this one was really hard :/  I wanted to put it into a function that can be called everywhere a change event needed to be emitted so that I could include the conditionals all in one place as opposed to spreading that out in all the calling locations.  I'll think on it some more, but that was the best I could come up with in the moment.

Reply:
`emitChangeEventIfNecessary()` but seems kinda long.

Reply:
`handleChangeEvent()`?